src/components/Observations/ObservationViews.js

```diff
@@ -22,31 +27,64 @@ import ObsListHeader from "./ObsListHeader";
 import UploadProgressBar from "./UploadProgressBar";
 import UploadPrompt from "./UploadPrompt";
 
+const { useRealm } = RealmContext;
+
 type Props = {
-  loading: boolean,
-  localObservations: Object,
-  testID: string,
+  testID?: string,
   taxonId?: number,
-  mapHeight?: number,
-  handleEndReached?: Function,
-  syncObservations?: Function
+  mapHeight?: number
 }
 
 const ObservationViews = ( {
-  loading,
-  localObservations,
   testID,
   taxonId,
-  mapHeight,
-  handleEndReached,
-  syncObservations
+  mapHeight
 }: Props ): Node => {
+  const localObservations = useLocalObservations( );
+  const realm = useRealm( );
   const [view, setView] = useState( "list" );
   const navigation = useNavigation( );
   const { name } = useRoute( );
   const isLoggedIn = useLoggedIn( );
   const { observationList, unuploadedObsList } = localObservations;
   const numOfUnuploadedObs = unuploadedObsList?.length;
+
+  const currentUser = realm.objects( "User" ).filtered( "signedIn == true" )[0];
+  const [idBelow, setIdBelow] = useState( null );
+
+  const params = {
+    user_id: currentUser?.id,
+    per_page: 10,
+    fields: Observation.FIELDS
+  };
+
+  if ( idBelow ) {
+    // $FlowIgnore
+    params.id_below = idBelow;
+  } else {
+    // $FlowIgnore
+    params.page = 1;
+  }
+
+  const {
+    data: observations,
+    isLoading
+  } = useAuthenticatedQuery(
+    ["searchObservations", idBelow],
+    optsWithAuth => searchObservations( params, optsWithAuth ),
+    {
+      keepPreviousData: true
+    }
+  );
+
+  const handleEndReached = oldestId => setIdBelow( oldestId );
```

Comment:
This is just getting called once, so is there a need for it to be its own function? IMO just doing this work in `onEndReached` below means one fewer jump when trying to understand what's happening.